README.md

```diff
@@ -2,41 +2,64 @@
 
 [![Build Status](https://dev.azure.com/sitecorejss/sitecore-jss-ci/_apis/build/status/Sitecore.jss?branchName=master)](https://dev.azure.com/sitecorejss/sitecore-jss-ci/_build/latest?definitionId=1?branchName=master) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)](contributing.md#pull-requests)
 
-This repository contains source code for all Sitecore JSS packages as well as samples to help you get started using Sitecore JSS.
+This repository contains source code for all Sitecore JSS packages and sample applications to help you get started using Sitecore JavaScript Services (JSS).
 
-Below is a list of samples included in this repo:
+This repository contains the following sample applications:
 
-1. [React](/samples/react)
+1. [Next.js](/samples/nextjs)
 
-   Built using [create-react-app](https://github.com/facebook/create-react-app) as a starting point. Supports out-of-the-box Sitecore functionalities (i18n, routing, GraphQL, set &lt;title&gt; tags) and demonstrates how to work with various Sitecore data types, placeholder nesting and wrapping techniques, and other helpful JSS patterns. The application code is heavily commented, but [documentation also exists](http://jss.sitecore.net/docs/client-frameworks/react/sample-app).
+   The Next.js-based sample application demonstrates common use cases. It was built using [create-next-app](https://nextjs.org/docs/api-reference/create-next-app) as a starting point. 
```

Comment:
I don't think I would mention `create-react-app` here. Though the react sample was indeed based on create-react-app, that was years ago and it may be very different today. We used the existing react sample and the [Next.js starter](https://github.com/vercel/next-learn-starter/tree/master/learn-starter) as references